src/Jwt.ts

```diff
@@ -3,17 +3,43 @@ import * as atlassianJwt from 'atlassian-jwt';
 import { AuthError, AuthErrorCode } from './AuthError';
 import { ConnectJwt } from './types';
 
+/**
+ * Checks whether the algorithm is asymmetric. Currently, only RS256 is supported.
+ */
+export function isAsymmetricAlgorithm(alg?: string): boolean {
+  return alg === atlassianJwt.AsymmetricAlgorithm.RS256;
+}
+
 /**
  * Decodes a Connect JWT without verifying its authenticity.
  */
 export function decodeUnverifiedConnectJwt(rawConnectJwt: string): ConnectJwt {
   try {
-    return atlassianJwt.decodeSymmetric(
-      rawConnectJwt,
-      '',
-      atlassianJwt.SymmetricAlgorithm.HS256,
-      true
-    );
+    const alg = atlassianJwt.getAlgorithm(rawConnectJwt) as string;
```

Comment:
For the sake of convenience, the decoding function supports both algorithms and also returns the algorithm with the payload so the request or installation verification functions can determine whether it is a signed install.